tests/unit/test_depth_anything.py

```
@@ -0,0 +1,233 @@
+
+        assert chw.dtype == np.float32
+        assert chw.shape[0] == 3
+        h, w = chw.shape[1], chw.shape[2]
+        assert h % 14 == 0 and w % 14 == 0
+        assert h >= 518 and w >= 518  # lower-bound resize
+        assert ratio == 1.0
+        # Normalized to ~[0, 1] (divided by 255). cv2.INTER_CUBIC can overshoot
+        # the [0, 1] range slightly, so allow a small margin rather than a hard
+        # clamp; the model applies ImageNet normalization next regardless.
+        assert np.isfinite(chw).all()
+        assert chw.min() >= -0.5 and chw.max() <= 1.5
```

Ruff (0.15.18)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)